### DISCUSSION TITLE
github.com/google/gopacket/pcap Erro installation

### DISCUSSION THREAD

**Discussion by Bugspiderlee** [projectdiscovery/naabu#945] [Q&A]
Hi Any easy fix for this after installation

usere-ubuntu:~$ go install -v github.com/projectdiscovery/naabu/v2/cmd/naabu@latest
github.com/google/gopacket/pcap
# github.com/google/gopacket/pcap
go/pkg/mod/github.com/google/gopacket@v1.1.19/pcap/pcap.go:30:22: undefined: pcapErrorNotActivated
go/pkg/mod/github.com/google/gopacket@v1.1.19/pcap/pcap.go:52:17: undefined: pcapTPtr
go/pkg/mod/github.com/google/gopacket@v1.1.19/pcap/pcap.go:64:10: undefined: pcapPkthdr
go/pkg/mod/github.com/google/gopacket@v1.1.19/pcap/pcap.go:103:6: undefined: pcapBpfProgram
go/pkg/mod/github.com/google/gopacket@v1.1.19/pcap/pcap.go:110:7: undefined: pcapPkthdr
go/pkg/mod/github.com/google/gopacket@v1.1.19/pcap/pcap.go:268:33: undefined: pcapErrorActivated
go/pkg/mod/github.com/google/gopacket@v1.1.19/pcap/pcap.go:269:33: undefined: pcapWarningPromisc
go/pkg/mod/github.com/google/gopacket@v1.1.19/pcap/pcap.go:270:33: undefined: pcapErrorNoSuchDevice
go/pkg/mod/github.com/google/gopacket@v1.1.19/pcap/pcap.go:271:33: undefined: pcapErrorDenied
go/pkg/mod/github.com/google/gopacket@v1.1.19/pcap/pcap.go:748:14: undefined: pcapTPtr
go/pkg/mod/github.com/google/gopacket@v1.1.19/pcap/pcap.go:271:33: too many errors

**Comment by ehsandeep** [comment]
@Bugspiderlee see https://github.com/projectdiscovery/naabu?tab=readme-ov-file#prerequisite

**Comment by Mzack9999** [comment]
Run in your shell:
```
$ sudo apt install libpcap-dev
```
